### PR TITLE
chore(doc): Prep for version 0.1.0

### PR DESCRIPTION
**Pull request by ncknt**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -7,20 +7,35 @@ node {
...
         stage("Build image ${version}") {
             sh 'make build-docker'
         }
-
         if (env.BRANCH_NAME == "master") {
             stage("Push image") {
                 sh 'make push publish'
             }
+        } else {
+            def branchMatch = env.BRANCH_NAME =~ /^release-(0|[1-9]\d*)\.(0|[1-9]\d*)\.x$/
```
Comment (kevinawoo) [inline Jenkinsfile]: why not just do `.*`?

File: Jenkinsfile [hunk 2]
```diff
@@ -7,20 +7,35 @@ node {
...
+                def versionMatch = releaseVersion =~ /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(.+))?$+/
+                if (!versionMatch) {
+                    error("Incorrect version ${releaseVersion} defined in ./operator-version")
+                }
+                if (versionMatch.group(1) != branchMatch.group(1) || versionMatch.group(2) != branchMatch.group(2)) {
+                    error("Version ${releaseVersion} does not match branch it is being built on ${env.BRANCH_NAME}")
+                }
+                props.releaseVersion = releaseVersion
+                stage("Publish Version ${releaseVersion}") {
+                    sh "make push publishRelease RELEASE_VERSION=\"${releaseVersion}\""
```
Comment (kevinawoo) [inline Jenkinsfile]: hmmmmmmmm do we have to commit the `operator-version`? why not just pull it from the branch?

Comment (ncknt) [reply Jenkinsfile]: For patch releases.

File: Jenkinsfile (hunk 1 above)
Comment (ncknt) [reply Jenkinsfile]: To capture the version and check the version matches (= we didn't forget to update the version file)

**Pull request merged**